.gitmodules

```diff
@@ -58,9 +58,6 @@
 [submodule "auxil/out_ptr"]
 	path = auxil/out_ptr
 	url = https://github.com/soasis/out_ptr.git
-[submodule "auxil/spicy-plugin"]
-	path = auxil/spicy-plugin
-	url = https://github.com/zeek/spicy-plugin
 [submodule "auxil/spicy"]
 	path = auxil/spicy/spicy
```

Comment:
Could this move to be auxil/spicy by itself now?

Reply:
I'm hoping for that, too, but I believe it's an independent question, not directly related to this PR.

Reply:
Separate PR for this now: https://github.com/zeek/zeek/pull/3041